### PR TITLE
[Refactor][API2.0] Api ppl other

### PR DESCRIPTION
**Pull request by grimoire**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

API of ppl and sdk export.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdeploy/apis/ncnn/__init__.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 from mmdeploy.backend.ncnn import from_onnx as _from_onnx
-from mmdeploy.backend.ncnn import is_available, is_plugin_available
+from mmdeploy.backend.ncnn import is_available, is_customops_available
 from ..core import PIPELINE_MANAGER
 
 from_onnx = PIPELINE_MANAGER.register_pipeline()(_from_onnx)
 
-__all__ = ['is_available', 'is_plugin_available', 'from_onnx']
+__all__ = ['is_available', 'is_customops_available', 'from_onnx']
```
Comment (AllentDan) [inline mmdeploy/apis/ncnn/__init__.py:8]: not sure if `is_customops_available` is appropriate.

**Review by AllentDan — APPROVED** [review]
LGTM

File: mmdeploy/apis/ncnn/__init__.py (hunk 1 above)
Comment (grimoire) [reply mmdeploy/apis/ncnn/__init__.py:8]: Not sure too.

File: tools/deploy.py [hunk 2]
```diff
@@ -12,11 +12,11 @@
                            get_predefined_partition_cfg, torch2onnx,
                            torch2torchscript, visualize_model)
 from mmdeploy.apis.core import PIPELINE_MANAGER
+from mmdeploy.backend.sdk.export_info import export2SDK
```
Comment (RunningLeon) [inline tools/deploy.py:15]: should we update `tools/onnx2ncnn.py` and `tools/onnx2pplnn.py` as well?

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**